app/views/layouts/application.html.erb

```diff
@@ -16,36 +16,41 @@
   </head>
 
   <body>
-	
-	<nav class="navbar navbar-toggleable-sm navbar-fixed-top navbar-light bg-faded">
-		<div class="container">
-			<a class="navbar-brand" href="#">Star Wars</a>
-		  <div class="navbar" id="navbarNav">
-		    <ul class="nav nav-pills">
-		      <% if user_signed_in? %>
-		      <%= render 'user_navbar' %>
-					<% else %>
-					<%= render 'guest_navbar' %>
-					<% end %>
-		    </ul>
-		  </div>
-		</div>
-	  
-	</nav>
+    
+    <nav class="navbar navbar-expand-md navbar-light bg-faded">
+        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+            <a class="navbar-brand" href="#">Star Wars</a>
+            <div class="collapse navbar-collapse" id="navbarText">
+                <ul class="navbar-nav mr-auto">
```

Comment:
Why don't you reuse the `user_signed_in?` if statement to display the user_navbar along with the other `ul` element?